src/vivarium_census_prl_synth_pop/components/population.py

```diff
@@ -60,6 +60,8 @@ def setup(self, builder: Builder):
             "entrance_time",
             "exit_time",
             "housing_type",
+            "guardian_1",
+            "guardian_2",
```

Comment:
i want to confirm: have the research team documented that a sim has a _max_ of 2 guardians?

Reply:
Yes.